Configuration/StandardSequences/python/ValidationCosmics_cff.py

```diff
@@ -10,8 +10,8 @@
 
 from Validation.Configuration.globalValidationCosmics_cff import *
 
-prevalidation = cms.Sequence(globalPrevalidationCosmics)
+prevalidationCosmics = cms.Sequence(globalPrevalidationCosmics)
```

Comment:
@franzoni , apologies for replying with a long delay on this.
This part of the validation machinery was not excercised in cosmics in a central workflow (was just used in workflows run privately, at least in recent past).
Anyhow MUO has people who joined the team in the past months that, after working on data certification for cosmics for 2016, plan to improve matrix workflows as 1307 .
That sequence (not included at present) is a good hook for them to start improving the monitoring in the workflow mentioned above.
(I've just discussed with @HuguesBrun that this is still in MUO plans)